Makefile

```diff
@@ -181,3 +181,6 @@ db-reset:
 
 libzauth:
 	$(MAKE) -C libs/libzauth install
+
+hie.yaml:
```

Comment:
I think this will refuse to re-generate the file once it's already there. Maybe it could be a PHONY target (instead of listing all relevant files as dependencies)?